includes/Core/Admin/Screens.php

```diff
@@ -329,11 +329,14 @@ private function get_screens() {
 						}
 					},
 					'render_callback'  => function( Context $context ) {
+						$authentication = new Authentication( $context );
```

Comment:
Please update `Screens` to accept `Authentication` as a new constructor dependency like the others - e.g. `Module` - (optionally only instantiating a new instance for tests). The "main" instance in `Plugin` should be passed through when instantiating `Screens`.

Reply:
Done - I have also updated the other place where authentication was being constructed.